cloudformation/content-authorisation-proxy.cf.json

```diff
@@ -146,6 +146,25 @@
                     }
                 },
                 {
+                    "PolicyName": "dynamoTable",
+                    "PolicyDocument": {
+                        "Version": "2012-10-17",
+                        "Statement": [
+                            {
+                                "Effect": "Allow",
+                                "Action": [
+                                    "dynamodb:PutItem",
+                                    "dynamodb:GetItem",
+                                    "dynamodb:UpdateItem",
+                                    "dynamodb:DeleteItem",
+                                    "dynamodb:BatchGetItem"
+                                ],
+                                "Resource": "arn:aws:dynamodb:eu-west-1:865473395570:table/cas-auth-PROD"
```

Comment:
There are CloudFormation functions which can fill in these variables, e.g.

```
!Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/cas-auth-PROD"
```

See: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html